bob/bob_task.py

```diff
@@ -0,0 +1,22 @@
+# Task from https://exercism.org/tracks/python/exercises/bob
+
+import re
+
+
+def response(hey_bob):
+    if hey_bob.strip() == "":
+        return 'Fine. Be that way!'
```

Comment:
Please choose one rule regarding quotation marks and use it everywhere in the code